djangocms_navigation/static/djangocms_navigation/js/navigation-tree-admin.js

```diff
@@ -132,199 +132,203 @@ Original code found in treebeard-admin.js
 
         $body = $('body');
 
-        // Activate all rows for drag & drop
-        // then bind mouse down event
-        $('td.drag-handler span').addClass('active').bind('mousedown', function (evt) {
-            $ghost = $('<div id="ghost"></div>');
-            $drag_line = $('<div id="drag_line"><span></span></div>');
-            $ghost.appendTo($body);
-            $drag_line.appendTo($body);
+        // Drag/drop only can be disabled. This ensures collapsible links are still activated.
+        // This is disabled when viewing a preview of a menu.
+        if ($('#disable-drag-drop').val() !== "1") {
```

Comment:
Note: this appears as a much bigger change than it is, it is simply adding this conditional which required the rest of the code to be outdented - the actual code is unchanged.